package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "wpcom",
   "description": "Official JavaScript library for the WordPress.com REST API",
-  "version": "4.7.0",
+  "version": "4.7.1",
```

Comment:
If we're following [SemVer](http://semver.org/), the changes here seem much more than simple bug fixes, and might justify the minor version bump.